### DISCUSSION TITLE
[Feature] config file

### DISCUSSION THREAD

**Discussion by kai-arsenault** [arabcoders/watchstate#271] [General] [answered]
It would be really great if there was a config file to add and to manage backends. It's a little tiring to go through all the questions it would be a lot nicer just to be able to put it into a config file like this...

backendname:
    type: plex
    url: domain:32400
etc...

**Comment by arabcoders** [comment, marked as answer]
Hi,

this is already possible by editing the `config/servers.yaml` file.
